src/components/Voters/Body/index.tsx

```diff
@@ -109,37 +118,37 @@ const Body: React.FC<BodyVoters> = ({ searchValue }) => {
       {edenMembers?.map(delegate => (
         <DelegateItem
           key={delegate[1].name}
+          actionItemStyles={classes.itemActionStyle}
           text={
-            delegate.vote
-              ? `${t('voters.voteFor')} ${delegate?.vote?.length} `
-              : t('voters.noVoting')
+            delegate?.vote?.state !== eosioUtil.VoteState.ForProxy
+              ? delegate?.vote?.state === eosioUtil.VoteState.NoVoting
+                ? t('voters.noVoting')
+                : t('voters.voteByOther')
+              : `${t('voters.voteFor')} ${String(delegate?.vote?.amount)} `
           }
           name={delegate[1].name}
           image={`https://ipfs.io/ipfs/${delegate?.info?.image}`}
```

Comment:
Eden has its own IPFS gateway which may be better to use:

https://eden-genesis.mypinata.cloud/ipfs/

Reply:
maybe even something worth configuring as an ENV var